### PR TITLE
Update nouveautes.html

### PR DESCRIPTION
**Pull request by agarrone**
Add septembre, octobre

### CODE REVIEW

File: pages/nouveautes.html [hunk 1]
```diff
@@ -37,6 +37,51 @@ <h2 class="fr-mt-4w">Développements à venir</h2>
...
+		</div>
+		<h3>Lancement de l'Open Data University</h3> 
+		<div class="markdown">
+		<p>L’Open Data University est un dispositif imaginé par l’association Latitudes et soutenu par la DINUM
+		pour une année d’expérimentation sur l’année scolaire 2022/2023. 
+		<a href="https://www.data.gouv.fr/fr/pages/odu/home/"></a>Découvrez l’Open Data University.</p>
+		</div>
+		<h3>Mise en place d'un Captcha souverain </h3> 
+		<div class="markdown">
+		<p>Pour Pour lutter contre la création de faux comptes utilisateur nous avons déployé CaptchEtat</p>
```
Comment (maudetes) [inline pages/nouveautes.html:82]:
```suggestion
		<p>Pour lutter contre la création de faux comptes utilisateur nous avons déployé CaptchEtat</p>
```

**Review by maudetes — APPROVED** [review]
~Looks good to me!~ -> woops some fixes to apply first :)

Do we want to add links to the different services/pages we mention?

File: pages/nouveautes.html [hunk 2]
```diff
@@ -37,6 +37,51 @@ <h2 class="fr-mt-4w">Développements à venir</h2>
...
+		<p>Pour faciliter l’exploitation des données, nous avons profondément revu la prévisualisation et l’exploration de données sur la plateforme. Nous attendons vos retours avec impatience !</p>
+		</div> 
+		<h3>Mise en place d'explore.data.gouv.fr/prix-carburants </h3> 
+		<div class="markdown">
+		<p>Nous disposons de nombreuses briques techniques ouvertes, mutualisables et réutilisables qu’il est possible de mobiliser pour proposer des visualisations de données sur une diversité de thématiques comme le logement, l’aménagement du territoire, l’éducation ou encore l’énergie.
+		En guise de démonstration, nous les avons appliquées à un exemple d’actualité en les utilisant sur les données des prix des carburants</p>
+		</div>
+		<h3>Publication d'une nouvelle page inventaire sur les données relative à l'énergie</h3> 
+		<div class="markdown">
+		<p>Cette page a pour vocation de référencer les principaux jeux de données relatifs à la thématique “énergie” disponibles sur data.gouv.fr. Nous avons également publié une série d'article sur le sujet.</p>
```
Comment (maudetes) [inline pages/nouveautes.html:72]:
```suggestion
		<p>Cette page a pour vocation de référencer les principaux jeux de données relatifs à la thématique “énergie” disponibles sur data.gouv.fr. Nous avons également publié une série d'articles sur le sujet.</p>
```

File: pages/nouveautes.html [hunk 3]
```diff
@@ -37,6 +37,51 @@ <h2 class="fr-mt-4w">Développements à venir</h2>
...
+		<h3>Mise en place d'explore.data.gouv.fr </h3> 
+		<div class="markdown">
+		<p>Pour faciliter l’exploitation des données, nous avons profondément revu la prévisualisation et l’exploration de données sur la plateforme. Nous attendons vos retours avec impatience !</p>
+		</div> 
+		<h3>Mise en place d'explore.data.gouv.fr/prix-carburants </h3> 
+		<div class="markdown">
+		<p>Nous disposons de nombreuses briques techniques ouvertes, mutualisables et réutilisables qu’il est possible de mobiliser pour proposer des visualisations de données sur une diversité de thématiques comme le logement, l’aménagement du territoire, l’éducation ou encore l’énergie.
+		En guise de démonstration, nous les avons appliquées à un exemple d’actualité en les utilisant sur les données des prix des carburants</p>
+		</div>
+		<h3>Publication d'une nouvelle page inventaire sur les données relative à l'énergie</h3>
```
Comment (maudetes) [inline pages/nouveautes.html:70]:
```suggestion
		<h3>Publication d'une nouvelle page inventaire sur les données relatives à l'énergie</h3> 
```

File: pages/nouveautes.html [hunk 4]
```diff
@@ -37,6 +37,51 @@ <h2 class="fr-mt-4w">Développements à venir</h2>
...
+		</div>
+		<h3>Publication d'une nouvelle page inventaire sur les données relative à l'énergie</h3> 
+		<div class="markdown">
+		<p>Cette page a pour vocation de référencer les principaux jeux de données relatifs à la thématique “énergie” disponibles sur data.gouv.fr. Nous avons également publié une série d'article sur le sujet.</p>
+		</div>
+		<h3>Lancement de l'Open Data University</h3> 
+		<div class="markdown">
+		<p>L’Open Data University est un dispositif imaginé par l’association Latitudes et soutenu par la DINUM
+		pour une année d’expérimentation sur l’année scolaire 2022/2023. 
+		<a href="https://www.data.gouv.fr/fr/pages/odu/home/"></a>Découvrez l’Open Data University.</p>
```
Comment (maudetes) [inline pages/nouveautes.html:78]:
```suggestion
		<a href="https://www.data.gouv.fr/fr/pages/odu/home/">Découvrez l’Open Data University.</a></p>
```

**Comment by agarrone** [comment]
> Do we want to add links to the different services/pages we mention?

Thanks! I added the link to the relevant page

**Pull request merged**